src/test/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibraryUnitTest.java

```diff
@@ -19,9 +18,170 @@
 import static org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.*;
 
 /**
- * Tests for the Wellformed read filter. .
+ * Tests for the read filter library.
  */
-public final class WellformedReadFilterUnitTest extends ReadFilterTest {
+public final class ReadFilterLibraryUnitTest {
+    private static final int DEFAULT_READ_GROUP_COUNT = 5;
+    private static final String DEFAULT_READ_GROUP_PREFIX = "ReadGroup";
+    private static final String DEFAULT_PLATFORM_UNIT_PREFIX = "Lane";
+    private static final String DEFAULT_PLATFORM_PREFIX = "Platform";
+    private static final int DEFAULT_CHROMOSOME_COUNT = 1;
+    private static final int DEFAULT_CHROMOSOME_START_INDEX = 1;
+    private static final int DEFAULT_CHROMOSOME_SIZE = 1000;
+
+    private final int groupCount;
+    private SAMFileHeader header;
+
+    /**
+     * Construct a read filter test with the default number of groups
+     *  ({@link #DEFAULT_READ_GROUP_COUNT}.
+     */
+    @SuppressWarnings("unused")
+    public ReadFilterLibraryUnitTest() {
+        this.groupCount = DEFAULT_READ_GROUP_COUNT;
+    }
+
+    /**
+     * Returns the mock-up SAM file header for testing.
+     *
+     * @throws IllegalStateException if the header was not initialized
+     *          invoking {@link #beforeClass()}
+     * @return never <code>null</code>
+     */
+    private SAMFileHeader getHeader() {
+        if (header == null) {
+            throw new IllegalArgumentException(
```

Comment:
This should create the header if there isn't one.  Throwing an exception probably made sense in the old usage, but not when it's only this specific class.